libevmasm/CommonSubexpressionEliminator.h

```diff
@@ -154,11 +155,11 @@ class CSECodeGenerator
 	/// Current height of the stack relative to the start.
 	int m_stackHeight = 0;
 	/// If (b, a) is in m_requests then b is needed to compute a.
-	std::multimap<Id, Id> m_neededBy;
+	std::unordered_multimap<Id, Id> m_neededBy;
```

Comment:
Can this make compilation in any way depend on the specific hash function?

Reply:
Taking into consideration, code does not rely on the order of the elements in this container, hashing function should have no impact on the output. But, I can imagine that it can have an impact on the performance in case of a poor hashing algorithm.